.github/workflows/verifyPackage.yml

```
@@ -0,0 +1,50 @@
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          
+      - name: Install dependencies
+        run: |
+          if [ ! -f package-lock.json ]; then
+            echo "::warning::Lockfile tidak ada. Membuat yang baru..."
+            npm install --package-lock-only
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)